packages/lib/ren-tx/src/machines/deposit.ts

```diff
@@ -14,6 +13,19 @@ export interface DepositMachineContext {
     deposit: GatewayTransaction;
 }
 
+const largest = (x?: number, y?: number): number => {
```

Comment:
Are these ever negative? If not would `Math.max(x || 0, y || 0)` do the same thing?